src/js/lib/DOMTranslation.ts

```diff
@@ -976,8 +986,10 @@ class DOMTranslation {
       }
     }
     for (const range of translationRanges) {
+      const isSeo = this.isTranslatableElementSeo(range.startMarker.parentElement.tagName);
```

Comment:
Var vienkārši: `TranslationElementCandidates.get(range.startMarker.parentElement.tagName)?.type === TranslatableItemType.ELEMENT_SEO`

Reply:
izņēmu to isTranslatableElementSeo metodi